docs/conf.py

```diff
@@ -53,13 +53,21 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-#
 html_theme = 'sphinx_rtd_theme'
 
+# Adds a logo to the navbar.
+html_logo = '../docs/liquidprompt_emblem.svg'
```

Comment:
Shouldn't need to go up and back down:
```suggestion
html_logo = 'liquidprompt_emblem.svg'
```

Reply:
:facepalm: